src/feedback/components/FeedbackList.vue

```diff
@@ -22,7 +22,9 @@
       <FeedbackItem
         v-for="feedbackitem in feedback"
         :key="feedbackitem.id"
+        :ref="refFor(feedbackitem.id)"
```

Comment:
Seems we need to create a ref for every item, even if we're just interested one...